TopQuarkAnalysis/TopEventProducers/src/PseudoTopProducer.cc

```diff
@@ -0,0 +1,578 @@
+#include "TopQuarkAnalysis/TopEventProducers/interface/PseudoTopProducer.h"
+
+#include "CommonTools/Utils/interface/PtComparator.h"
+
+#include "RecoJets/JetProducers/interface/JetSpecific.h"
+#include "fastjet/ClusterSequence.hh"
+
+using namespace std;
+using namespace edm;
+using namespace reco;
+
+PseudoTopProducer::PseudoTopProducer(const edm::ParameterSet& pset):
+  leptonMinPt_(pset.getParameter<double>("leptonMinPt")),
+  leptonMaxEta_(pset.getParameter<double>("leptonMaxEta")),
+  jetMinPt_(pset.getParameter<double>("jetMinPt")),
+  jetMaxEta_(pset.getParameter<double>("jetMaxEta")),
+  wMass_(pset.getParameter<double>("wMass")),
+  tMass_(pset.getParameter<double>("tMass"))
+{
+  finalStateToken_ = consumes<edm::View<reco::Candidate> >(pset.getParameter<edm::InputTag>("finalStates"));
```

Comment:
Basically, that's just for a cosmetics :-)
(Personally, I usually not use the init list but put them since I declared those variables as const ones.)
Anyway I can update them to be initialized in the same manner if it is better to fit with TQAF coding style.

Regarding on the  consumes ones, destructors are not necessary. It just let the framework to know which objects to be used.